.git-blame-ignore-revs

```diff
@@ -0,0 +1,17 @@
+##
+# All "yarn fix" commit revs should be included in this file.
+#
+# Add to your local configuration:
+#   git config --global blame.ignoreRevsFile .git-blame-ignore-revs
+# 
+# Verify all "yarn fix" commits are present in this file:
+#   git log --all -i --grep='yarn\sfix' --pretty=%H
+##
+
+8b0bc2bc945073f89b05f9b064b84f11af6299c3
+17372b17e0b91ee827e339f889a3ac19253a3c73
+13b995842ad952740304372d174a733d65c03640
+a7ba8d46e6f2fc3041af03c0f30b4d09ace46f44
+99c56fb565b1886ad756aa296f3c0170602529fc
+da8a3a1eb5cb5aff987867631b47a36055aaae5a
+7bbc8c715947383a6240a5d17512b52dc42cb958
```

Comment:
Please add:

```
# 2018-02-13 Initial prettier formatting
c990e091589dddfbf56c3bdbe166f25b22d53d00
```

Putting a date comment above each of the other hashes might helpful too, since otherwise this is pretty opaque.